RefactoringPracticeI/GameInventory/UnitTest/GameInventoryTest.cs

```diff
@@ -7,33 +7,78 @@ namespace RefactoringExerciseI.UnitTest
     public class GameInventoryTest
     {
         [Test]
-        public void BackstagePasses()
+        [TestCase(5, 0, 4, 3)]
+        [TestCase(0, 45, -1, 0)]
+        [TestCase(5, 50, 4, 50)]
+        [TestCase(8, 0, 7, 2)]
+        public void BackstagePasses(int startingSellIn, int startingQuality, int expectedSellIn, int expectedQuality)
         {
-            // ToDo
+            IList<Item> Items = new List<Item>();
+            ItemFactory backstagePassesFactory = new BackstagePassesFactory();
+
+            Items.Add(backstagePassesFactory.CreateItem("Backstage passes to a Pokemon Gym concert", startingSellIn, startingQuality));
```

Comment:
Subjective: wherever you feel something is being repeated too much, consider turning it into a constant, not a big deal, but at least we get rid of typos issues.